adapter/noop2/noop.go

```diff
@@ -82,37 +84,41 @@ func (builder) ConfigureQuotaHandler(map[string]*quota.Type) error {
 
 ////////////////// Runtime Methods //////////////////////////
 
-var cacheInfo = adapter.CacheabilityInfo{
+var checkResult = adapter.CheckResult{
+	Status:        rpc.Status{Code: int32(rpc.OK)},
 	ValidDuration: 1000000000 * time.Second,
 	ValidUseCount: 1000000000,
 }
+var reportResult = adapter.ReportResult{
```

Comment:
We don't need ReportResult as a thing. "error" is all we need for the Report case.